sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2858,6 +2858,14 @@ object SQLConf {
     .stringConf
     .createWithDefault("")
 
+  val JDBC_MYSQL_VERSION =
+    buildConf("spark.sql.jdbc.mysql.version")
```

Comment:
what if users want to connect to both mysql 5 and 8? I think it's better to make it a catalog option, not a session config.